en/appendices/4-1-migration-guide.rst

```diff
@@ -79,6 +84,14 @@ Console
   ANSI escape codes for colours. See `no-color.org <https://no-color.org/>`__
   for more information.
 
+Database
+--------
+
+* ``AggregateExpression`` was added to represent aggregate SQL functions. ``FunctionsBuilder::aggregate()``
+  can be used to wrap new aggregate SQL functions.
+* Window function support was added for any aggregate expression. ``AggregateExpression`` wraps the window
+  expression making it easy to extend any instance with call chaining.
```

Comment:
Do we have usage docs for windowing functions? It would be good to add a section to the database query building docs on how to use these new features.

Reply:
I need to write a section on it now that it's ready.